build_tools/setup_helpers/extension.py

```diff
@@ -0,0 +1,113 @@
+import platform
+import subprocess
+from pathlib import Path
+
+from torch.utils.cpp_extension import (
+    CppExtension,
+    BuildExtension as TorchBuildExtension
+)
+
+__all__ = [
+    'get_ext_modules',
+    'BuildExtension',
+]
+
+_ROOT_DIR = Path(__file__).parent.parent.parent.resolve()
+_CSRC_DIR = _ROOT_DIR / 'torchtext' / 'csrc'
+_TP_BASE_DIR = _ROOT_DIR / 'third_party'
+_TP_INSTALL_DIR = _TP_BASE_DIR / 'build'
+
+
+def _get_eca(debug):
+    eca = []
+    if debug:
+        eca += ["-O0", "-g"]
+    return eca
+
+
+def _get_ela(debug):
```

Comment:
What about "-O3" etc. for non-debug builds?

Reply:
Sure we can do that. If you give me the list of typical flags that can go here, it would be helpful too.

Reply:
Let's start with O3.